recipes/nvidia-ml-py/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "nvidia-ml-py" %}
+{% set version = "11.450.51" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5aa6dd23a140b1ef2314eee5ca154a45397b03e68fd9ebc4f72005979f511c73
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+  run:
+    - python >=3.4
+  run_constrained:
+    - nvidia-ml ==9999999999
+    - pynvml ==9999999999
```

Comment:
Both package `nvidia-ml` and `pynvml` modify module `pynvml.py` in site-packges. They are in conflict with `nvidia-ml-py` (the official NVML Python Bindings maintained by NVIDIA).